BEIMA.Backend/Resources.resx

```diff
@@ -0,0 +1,132 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is a resx file that will allow us to consolidate api messages/error messages into one place.